test/integration/scheduler/queue_test.go

```diff
@@ -207,7 +209,9 @@ func TestCoreResourceEnqueue(t *testing.T) {
 		// All of them are expected to be unschedulable at first.
 		pods []*v1.Pod
 		// triggerFn is the function that triggers the event to move Pods.
-		triggerFn func(testCtx *testutils.TestContext) error
+		// It returns the map keyed with ClusterEvents to be triggered by this function,
```

Comment:
Is it a typo? "map keyed with ClusterEvents" -> "map key with ClusterEvents"

Reply:
No. I often see `keyed` and `valued` describing the map's key type and value type.

Reply:
ok